helper/fields/data.go

```diff
@@ -18,7 +18,20 @@ type FieldData struct {
 func (d *FieldData) Validate() error {
 	var result *multierror.Error
 
-	// Scan for missing required fields
+	// Check that either image or pod_manifest was passed.
+	_, podManifestExists := d.Raw["pod_manifest"]
+	_, imageExists := d.Raw["image"]
+
+	if !podManifestExists && !imageExists {
+		result = multierror.Append(result, fmt.Errorf(
+			"both field pod_manifest and image are missing, but only one is required",
```

Comment:
```go
result = multierror.Append(result, fmt.Errorf(`either field "pod_manifest" or "image" are required, but none was found`))
```

Reply:
Won't this break other callers of `FieldData.Validate()`. Other drivers call it and might not have any of "pod_manifest" nor "image".

Reply:
Good catch. But I'm not sure of a clean way to implement this check as the `Validate` method does not have driver information which would have helped to decide to do this check or not.

The other way I can think of is doing this check before calling the `Validate` function in `rkt.go` but that defeats the purpose of `Validate` and makes it hacky. Any thoughts on doing this in a better way?